### PR TITLE
Added tests and fix dbrestore command

### PR DESCRIPTION
**Pull request by ZuluPro**
For tests:
- Added file fixture: the same file raw, compressed, encrypted and both
- Added ``django.utils.six`` for deal with Python 2/3
- Use Bytes mode to file read/write (for Python 3)
- Skip tests in Python3, only those which didn't work
- Added possibility to launch a shell instead of tests

For source code:
- Use Bytes mode to file read/write (for Python 3)
- Changed ``print`` to self.stdout (again ;) )

### CODE REVIEW

**Comment by benjaoming** [comment]
Great work!! Could you do a merge with current master?

**Comment by ZuluPro** [comment]
I just made a ``git pull --rebase`` from your master.
It includes my commit without make a merge.

File: dbbackup/management/commands/dbrestore.py [hunk 1]
```diff
@@ -112,7 +107,7 @@ def uncompress_file(self, inputfile):
         outputfile = tempfile.SpooledTemporaryFile(
             max_size=500 * 1024 * 1024,
             dir=dbbackup_settings.TMP_DIR)
-        zipfile = gzip.GzipFile(fileobj=inputfile, mode="r")
+        zipfile = gzip.GzipFile(fileobj=inputfile, mode="rb")
```
Comment (benjaoming) [inline dbbackup/management/commands/dbrestore.py]: that looks like an important catch!

**Comment by benjaoming** [comment]
Awesome work!

**Pull request merged**